runtime/compiler/p/codegen/J9CodeGenerator.cpp

```diff
@@ -94,7 +94,7 @@ J9::Power::CodeGenerator::initialize()
 
    if (comp->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P8) && comp->target().cpu.supportsFeature(OMR_FEATURE_PPC_HAS_VSX) &&
       comp->target().is64Bit() && !comp->getOption(TR_DisableFastStringIndexOf) &&
-      !TR::Compiler->om.canGenerateArraylets())
+      !TR::Compiler->om.canGenerateArraylets() && !TR::Compiler->om.isOffHeapAllocationEnabled())
```

Comment:
i think Jimmy already took care of off-heap case in the new indexOf impl, but we can take care of this disabling later double-checking the code.

Reply:
Added this to the list of later offheap todos